test/show_test.rb

```diff
@@ -0,0 +1,44 @@
+require './lib/character'
+require './lib/show'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+
+class ShowTest < Minitest::Test
+
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+
+  end
+
+  def test_it_exists
+
+    assert_instance_of Show, @knight_rider
+  end
+
+  def test_it_has_a_name
+
+    assert_equal "Knight Rider", @knight_rider.name
+  end
+
+  def test_it_has_a_creator
+
+    assert_equal "Glen Larson", @knight_rider.creator
+  end
+
+  def test_it_has_characters
+
+    assert_equal [@michael_knight, @kitt], @knight_rider.characters
+  end
+
+  def test_it_has_a_total_salary
+
+    assert_equal 2600000, @knight_rider.total_salary
+  end
+
+
+
+end
```

Comment:
Good testing in here, be sure to clean up extra blank lines in your methods and generally throughout the file as well for easier reading.